headless-server-commerce-parent/pom.xml

```diff
@@ -57,7 +57,7 @@
     <guava.version>33.4.0-jre</guava.version>
     <htmlcleaner.version>2.29</htmlcleaner.version>
     <json.version>20250107</json.version>
-    <tomcat.version>10.1.34</tomcat.version>
+    <tomcat.version>11.0.5</tomcat.version>
```

Comment:
we should stick to tomcat 10 as we do with CMCC 12, see https://github.com/CoreMedia/cms/commit/01b7d498cf96d4f9acfae97cee030ba6987742a6 - upgrade to tomcat 10.1.39 instead of 11.0.5

Reply:
funny, I had exceptions with 10.1.39 in the first place. but I just tried again and no errors. so i will go with  10.1.39